Jenkinsfile

```diff
@@ -186,7 +188,7 @@ def initializeEnvironment() {
   }
 
   sh label: 'Install unit test modules', script: '''#!/bin/bash -lex
-    pip install nose-ignore-docstring nose-exclude service_identity
+    pip install pynose nose-ignore-docstring nose-exclude service_identity
```

Comment:
Nice, interesting find!  It's at least a better short-term bridge until we can get onto pytest